src/utils/error.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * ISC License (ISC)
+ * Copyright 2018 aeternity developers
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+ *  REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+
+export class AeError extends Error {}
+
+export class InvalidKeypairError extends AeError { }
+
+export class UnAvailableAccountError extends AeError {
+  constructor (address: string) {
+    super(`Account for ${address} not available`)
+  }
+}
+export class UnknownAccountError extends AeError {
+  constructor () {
+    super('No account or wallet configured')
+  }
+}
+export class UnknownAccountTypeError extends AeError {
+  constructor (account: any) {
+    super(`Unknown account type: ${typeof account} (account: ${String(account)})`)

```

Comment:
```suggestion
    super(`Unknown account type: ${typeof account} (account: ${account})`)
```
won't it work the same?

Reply:
Works the same but the existing one is marked invalid by the linter. https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/restrict-template-expressions.md